internal/table/table.go

```diff
@@ -188,3 +219,25 @@ func pad(v string) string {
 	padded := strings.Repeat(" ", padding)
 	return padded + v + padded
 }
+
+func verticalRows(hdrs []string, rows []row) []row {
```

Comment:
Please undo all this stuff to the table package. As the ACs state the only change here is disregarding the headers.